src/tools/singlejar/token_stream_test.cc

```diff
@@ -86,9 +87,21 @@ TEST(TokenStreamTest, CommandFileLongPath) {
   const char *tempdir = getenv("TEST_TMPDIR");
   ASSERT_NE(nullptr, tempdir);
 
-  std::string command_file_path = singlejar_test_util::OutputFilePath(std::string(255, 'A'));
   std::wstring wpath;
   std::string error;
+  std::string command_file_path = singlejar_test_util::OutputFilePath(std::string(251, 'a'));
```

Comment:
For the record I meant something else, but this will do too.

(I meant creating a path whose directory components are all 8.3 style paths already. Bazel's JNI library shortens paths to 8.3 style when creating processes, because processes can only be created with short paths, and I thought `_wfopen` might, in theory, do the same, but I realize this is too paranoid to test for.)